Kernel/Storage/StorageDevice.cpp

```diff
@@ -30,6 +30,7 @@ StringView StorageDevice::class_name() const
 ErrorOr<size_t> StorageDevice::read(OpenFileDescription&, u64 offset, UserOrKernelBuffer& outbuf, size_t len)
```

Comment:
I know the parameter is called len (tbh it shouldn't be, but that's not introduced in this PR), but can we use the full word in the commit title/description?

Reply:
Sure, that sounds good